blocks/cards/cards.css

```diff
@@ -1 +1,432 @@
-/* TODO */
+.landing-page main .cards-wrapper {
+  width: 100%;
+}
+
+.landing-page main .cards-wrapper .cards.block {
+  max-width: 1440px;
+  margin: auto;
+  height: 720px;
+  margin-bottom: 40px;
+  overflow: hidden;
+}
+
+.landing-page main .cards-wrapper .cards.block .date-string {
+  display: none;
+}
+
+.landing-page main .cards-wrapper .cards.block .card-block-wrapper {
+  padding: 0 15px;
+  width: 100%;
+  height: 100%;
+  display: flex;
+  gap: 24px;
+}
+
+.landing-page main .cards-wrapper .cards.block .card-block-wrapper.reverse {
+  flex-direction: row-reverse;
+}
+
+.landing-page main .cards-wrapper .cards.block .card-block-wrapper .main-card,
+.landing-page main .cards-wrapper .cards.block .card-block-wrapper .small-card {
+  color: white;
+  text-decoration: none;
+  position: relative;
+  flex: 1 1 0px;
+}
+
+.landing-page main .cards-wrapper .cards.block .card-block-wrapper .main-card .image-bg {
+  position: absolute;
+  width: 100%;
+  height: 100%;
+  overflow: hidden;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.landing-page main .cards-wrapper .cards.block .card-block-wrapper .main-card .image-bg > img {
+  width: 100%;
+  height: 100%;
+  object-fit: cover;
+  transition: transform 3s;
+}
+
+.landing-page main .cards-wrapper .cards.block .card-block-wrapper .main-card:hover .image-bg > img {
+  transform: scale(1.2);
+}
+
+.landing-page main .cards-wrapper .cards.block .card-block-wrapper .main-card .main-text-wrapper {
+  background: linear-gradient(180deg, transparent 50%, #000);
+  position: relative;
+  width: 100%;
+  height: 100%;
+  padding: 40px;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-end;
+}
+
+.landing-page main .cards-wrapper .cards.block .card-block-wrapper .headline > * {
+  display: block;
+  color: #151517;
+  font-family: Gotham, arial, sans-serif;
```

Comment:
you can use the CSS var instead https://github.com/headwirecom/helix-sportsmagazine/blob/main/styles/styles.css#L15